recipes/pbzip2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.1.13" %}
+{% set major_version = "1.1" %}
+{% set sha1 = "f61e65a7616a3492815d18689c202d0685fe167d" %}
+{% set md5 = "4cb87da2dba05540afce162f34b3a9a6" %}
+
+package:
+  name: pbzip2
+  version: {{ version }}
+
+source:
+  fn: pbzip2-{{ version }}.tar.gz
+  url: https://launchpad.net/pbzip2/{{ major_version }}/{{ version }}/+download/pbzip2-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
```

Comment:
It configures how compilation occurs. See [`activate.sh`]( https://github.com/conda-forge/toolchain-feedstock/blob/9ed1c8d6b58e4186684ed1e72a167fcabf3a1383/recipe/activate.sh ) to get an idea of how this works. Basically this file is sourced before the build starts setting a bunch of useful environment variables to control compilation.